numpy/core/code_generators/generate_umath.py

```diff
@@ -502,6 +502,24 @@ def english_upper(s):
           TD(noobj),
           TD(O, f='npy_ObjectMin')
           ),
+'clip':
+    Ufunc(3, 1, ReorderableNone,
+          docstrings.get('numpy.core.umath.clip'),
+          'PyUFunc_DefaultTypeResolver',
+          TD(noobj),
+          ),
+'clip_above':
+    Ufunc(2, 1, ReorderableNone,
+          "Internal helper to implement clip with only a maximum",
+          'PyUFunc_DefaultTypeResolver',
+          TD(noobj),
+          ),
+'clip_below':
```

Comment:
Naming is hard. I indended this to mean "clip everything _that is_ below a certain value", but it could be equally read as "clip everything _to_ below a certain value".

Naming advice?

Reply:
Now moot, @mhvk argues these overloads provided no value.